cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
I think it will not pass our verify scripts, since it requires kubernetes-authors copyright.

I'm not sure how to solve this problem though - ideally it should go to vendor/ dir, but I'm not sure if this is possible.